crates/swc_typescript/src/lib.rs

```diff
@@ -1,14 +1,914 @@
-pub fn add(left: usize, right: usize) -> usize {
-    left + right
+#![allow(clippy::boxed_local)]
+
```

Comment:
Please add license comments for where this came from

Reply:
Did it, thanks!